internal/statesync/reactor.go

```diff
@@ -1006,14 +1006,26 @@ func (r *Reactor) waitForEnoughPeers(ctx context.Context, numPeers int) error {
 	startAt := time.Now()
 	t := time.NewTicker(100 * time.Millisecond)
 	defer t.Stop()
+	logT := time.NewTicker(time.Minute)
+	defer logT.Stop()
+	var iter int
 	for r.peers.Len() < numPeers {
+		iter++
 		select {
 		case <-ctx.Done():
 			return fmt.Errorf("operation canceled while waiting for peers after %s", time.Since(startAt))
 		case <-r.closeCh:
 			return fmt.Errorf("shutdown while waiting for peers after %s", time.Since(startAt))
 		case <-t.C:
 			continue
+		case <-logT.C:
+			r.Logger.Info("waiting for sufficient peers to start statesync",
+				"duration", time.Since(startAt).String(),
+				"target", numPeers,
+				"peers", r.peers.Len(),
+				"iters", iter,
```

Comment:
nit: showing the iterations isn't really that valuable of a piece of information when the user doesn't know how often the timer ticks. Far more useful to show the elapsed time waiting for peers

Reply:
It could still be helpful for knowing how often the timer has fired. The timestamp on the log will let you infer the timer if it matters—though I imagine for debugging the fact _that_ a tick happened is more important than when.